app/src/main/java/com/example/criminalintent/presentation/crime_list/CrimeListFragment.kt

```diff
@@ -51,16 +53,31 @@ class CrimeListFragment : Fragment() {
         observeData()
     }
 
+    override fun onCreateOptionsMenu(menu: Menu, inflater: MenuInflater) {
+        super.onCreateOptionsMenu(menu, inflater)
+        inflater.inflate(R.menu.fragment_crime_list, menu)
+    }
+
+    override fun onOptionsItemSelected(item: MenuItem): Boolean {
+        return when (item.itemId) {
+            R.id.new_crime -> {
+                val crime = Crime()
+                crimeListViewModel.addCrime(crime)
```

Comment:
This fragment displays a list of crimes. It doesn't have UI that allows for configuring a crime, so why have logic for that here?